src/Tribe/Editor/REST/V1/Endpoints/Single_Ticket.php

```diff
@@ -254,6 +255,16 @@ public function add_ticket( WP_REST_Request $request, $nonce_action ) {
 			);
 		}
 
+		$is_paypal_ticket = $provider instanceof Tribe__Tickets__Commerce__PayPal__Main || $provider instanceof \TEC\Tickets\Commerce\Module;
+
+		if ( $is_paypal_ticket && ( ! is_numeric( $body['price'] ) || (float)$body['price'] <= 0 )) {
```

Comment:
```suggestion
		$is_invalid_price = ( empty( $body['price']  ) || ! is_numeric( $body['price'] ) || (float) $body['price'] <= 0 );
		if (
			$is_paypal_ticket 
			&& $is_invalid_price
		) {
```

Reply:
Suggestion applied. Thanks for noticing this one!